.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
    - id: fix-encoding-pragma
    - id: check-docstring-first
 - repo: https://github.com/johann-petrak/licenseheaders.git
-  rev: 'master'
```

Comment:
A quick drive-by fix - `pre-commit` was upset that this was not set to a fixed revision, so unexpected changes could be brought in.